examples/generic-azure.yaml

```diff
@@ -164,12 +164,37 @@ groups:
             - flag: "udpu"
         remediation: |
           ## Remediation
-          It will be difficult to change the corosync MCAST transport to UCAST
-          on an existing cluster. Besides adding the 'transport: udpu' parameter one
-          would have to configure each node separatelly. The easiest way to
-          make the cluster UCAST it to bootstrap it. When creating the cluster
-          you would have to add the key -u, for example
-          $ crm cluster init -u ...
+          To change the corosync MCAST transport to UCAST edit the /etc/corosync/corosync.conf
+          as in the example
+          ```
+              max_messages: 20
+              interface {
+                  ringnumber: 0
+          -       bindnetaddr: 10.162.32.167
+          -       mcastaddr: 239.11.100.41
+                  mcastport: 5405
+                  ttl: 1
+              }
+          +   transport: udpu
+          ...
+          +nodelist {
+          +       node {
+          +               ring0_addr: 10.162.32.167
+          +               nodeid: 1
+          +       }
+          +
+          +       node {
+          +               ring0_addr: 10.162.32.89
+          +               nodeid: 2
+          +       }
+          +
+          +}
+          ```
+          1) In the totem section, in the interface subsection remove the
+          keys-value pairs **bindnetaddr** and **mcastaddr**.
+          2) In the totem section add key-value pair **transport: udpu**
+          3) Add section nodelist and subsections node for each nodes of the
+          cluster, where the **ring0_addr** is the IP address of the node.
 
           ## References
           - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
```

Comment:
Please always also reference SUSE guides where they exist.
In this case section 9.1.3 of https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/

Moreover I do not even find the UCAST recommendation at: https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
Actually that MSFT guide does not even seem to mention udpu explicitly either. Can you have a second look to see if my eyes are missing these details, or if you looks a a different MSFT link? We really should avoid making incorrect claims about what sources recommend, unless I missed something or it was changed of course.